content/collections/_index.fr.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Collections
+html_description: Parcourir les collections Open Terms Archive.
+---
+
+Nous établissons des partenariats pour suivre les conditions d'utilisation de nombreuses industries, dans plusieurs langues et juridictions.
```

Comment:
```suggestion
Les collections sont créées et maintenues par des groupes ayant un intérêt commun pour suivre les termes de services dans des industries, langues et juridictions spécifiques. Les collections permettent de collaborer tant sur la collecte que l'analyse des données. Pour exploiter les données Open Terms Archive, commencez par identifier la collection qui correspond à vos centres d’intérêts et rejoignez-la, ou créez-en une nouvelle si aucune ne correspond.
```

Reply:
Same here `conditions d'utilisations de services` instead of `termes de services`